tools/src/main/scala/scala/scalanative/interflow/MergeProcessor.scala

```diff
@@ -101,8 +101,8 @@ final class MergeProcessor(insts: Array[Inst],
           def mergeLocal(local: Local, value: Val): Unit = {
             val values = mutable.UnrolledBuffer.empty[Val]
             states.foreach { s =>
-              if (s.locals.contains(local)) {
-                values += s.locals(local)
+              s.locals.get(local).foreach { l =>
```

Comment:
Also we can simply it the following:
```suggestion
              s.locals.get(local).foreach(values += _ )
```

Reply:
Applied in 81ba80c.